tuna/update_golden.py

```diff
@@ -109,6 +109,31 @@ def get_fdb_query(dbt):
   return query
 
 
+class Dummy:  # pylint: disable=too-few-public-methods
+  """empty object"""
+
+
+def get_fdb_entries(dbt):
+  """! Compose query to get all fdb entries for the session, performs better than get_fdb_query"""
+  with DbSession() as session:
+    fdb_attr = [
+        "config", "solver", "session", "fdb_key", "params", "kernel_time",
+        "workspace_sz", "alg_lib", "opencl", "kernel_group"
+    ]
+    attr_str = ','.join(fdb_attr)
+    query = f"select {attr_str} from {dbt.find_db_table.__tablename__}"\
+            f" where valid=1 and session={dbt.session_id};"
+    ret = session.execute(query)
+    entries = []
+    for row in ret:
+      entry = Dummy()
+      for i, col in enumerate(fdb_attr):
+        setattr(entry, col, row[i])
+      entries.append(entry)
```

Comment:
Correct, this replaces the 'get_fdb_query' function and creates a namespace instead of a sqlalchemy orm. I discovered that this made a difference of 2 hours when handling the full gfx90878 dataset (shorter time was 15 min).